docs/user/aws/install.md

```diff
@@ -60,5 +63,110 @@ The OpenShift console is available via the kubeadmin login provided by the insta
 
 ![OpenShift web console](images/install_console.png)
 
+## Example: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. This example makes use of and reference to several 
+CloudFormation templates located in this repository.
+
+### Create Ignition Configs
+
+The machines will be manually started. Therefore, it is required to generate the bootstrap and machine ignition configs 
+and store them for later steps.
+
+```console
+[~]$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain openshiftcorp.com
```

Comment:
You've mentioned that you own this, but I'd still prefer `example.com`.  Or follow https://cloud.openshift.com/clusters/install and use `openshift.example.com`.  Either way, we don't have to worry about what happens if/when your ownership of the non-example domain lapses.

Reply:
Yes, fixed this. I started to on the initial push, but didn't want to conflict with your other PR/merge.

Reply:
Fixed.